modules/virt-features-for-storage-matrix.adoc

```diff
@@ -118,10 +123,11 @@ ifeval::["{VirtVersion}" >= "2.5"]
 endif::[]
 --
 endif::[]
-ifdef::virt-features-for-storage,virt-importing-rhv-vm[]
+ifdef::virt-features-for-storage[]
```

Comment:
@bgaydosrh  Here is the problem. You deleted the ifdef for `virt-importing-rhv-vm`. That removed the callouts in the RHV section.

Reply:
Fixed, thanks @apinnick and apologies.